Resources/Prototypes/Entities/Constructible/Power/lathe.yml

```diff
@@ -135,6 +135,7 @@
       - LightBulb
       - SheetSteel
       - GlassStack
+      - SheetPlastic
       - CableStack
```

Comment:
At least try and order it :\

Reply:
Yeah I will change it in a few hours also didn’t think the order mattered when I looked at them